README.md

```diff
@@ -60,42 +60,40 @@ We want to keep type safety as strict as possibble. In order to do that, we upda
 
 We want to format our code automatically. So, we need to install Prettier.
 
-```
+```sh
 yarn add prettier --dev
 ```
 
-```json
-# .prettierrc
+`.prettierrc`
 
+```jsonc
 {
   "printWidth": 100,
   "singleQuote": true,
   "trailingComma": "all"
 }
 ```
 
-```
-# .prettierignore
+`.prettierignore`
 
+```ignore
 build
-coverage
 ```
 
 Also, we want to enable format on save on VSCode.
 
-```json
-# .vscode/settings.json
+`.vscode/settings.json`
 
+```json
 {
   "editor.formatOnSave": true,
-  "prettier.eslintIntegration": true
 }
 ```
 
 Finally, we update `package.json` with related format scripts.
 
-```
-"format:ts": "prettier --write 'src/**/*.{ts,tsx}' && eslint --fix .",
```

Comment:
We can update this like `eslint --fix 'src/**/*.{ts,tsx}'` instead of remove completely

Reply:
At this stage `eslint` is not installed yet, I have updated this at step 5.